tests/cli/commands/test_scheduler_command.py

```diff
@@ -73,6 +73,23 @@ def test_skip_serve_logs(self, mock_process, mock_scheduler_job, executor):
             scheduler_command.scheduler(args)
             with pytest.raises(AssertionError):
                 mock_process.assert_has_calls([mock.call(target=serve_logs)])
+    @mock.patch.dict(
```

Comment:
How can I change the configuration value `[database][check_migrations]` in the tests? This did not work, and neither did line 89.

Reply:
I think `conf_vars` should work

Reply:
Yeah, with the `with conf_vars` you have looks right. Let me take a look and see.

Reply:
Ahha, the problem is https://github.com/apache/airflow/pull/29714/files#r1115484690

Because the value is set to `"False"`, i.e. a string, which is truthy. Change it to `getboolean` and it works as expected